src/graphql/root/mutation/user-logout.ts

```
@@ -0,0 +1,39 @@
+import { GT } from "@graphql/index"
+
+import AuthToken from "@graphql/types/scalar/auth-token"
+import AuthTokenPayload from "@graphql/types/payload/auth-token"
+import { logout } from "@app/auth"
+import { mapAndParseErrorForGqlResponse } from "@graphql/error-map"
+
+const UserLogoutInput = GT.Input({
+  name: "UserLogoutInput",
+  fields: () => ({
+    authToken: {
+      type: GT.NonNull(AuthToken),
+    },
+  }),
+})
+
+const UserLogoutMutation = GT.Field<{
+  input: {
+    authToken: SessionToken | InputValidationError
+  }
+}>({
+  extensions: {
+    complexity: 120,
+  },
+  type: GT.NonNull(AuthTokenPayload),
+  args: {
+    input: { type: GT.NonNull(UserLogoutInput) },
+  },
+  resolve: async (_, args) => {
+    const { authToken } = args.input
+    if (authToken instanceof Error) return
```

you should return an error like line 34 or 35